### ISSUE TITLE
Feature: hide balance

### ISSUE THREAD

**Issue by amassarwi** [decred/decrediton#2730]
This issue brings to discussion the famous _hide balance_ feature which nowadays available in many financial services.

Do we want that in decrediton ? 🤔

**Comment by exitus1** [comment]
Yes, I would love this feature as an option in both Decrediton and Mobile.

Also displaying your balance in terms of Fiat or BTC would be helpful.

**Comment by DCR-El-Capitan** [comment]
100% think this feature would be a great value add - have been hoping for something of this sort since I first hopped into the wallet!
